### ISSUE TITLE
Error when claiming drop rewards

### ISSUE THREAD

**Issue by delta-whiplash** [cyberofficial/TTVDropBot#17]
Hello I have this error when the bot want to claim the twitch drop

GQL RESPONSE ERROR! DropsPage_ClaimDropRewards Request Failed... Retrying in 60 seconds... Try: 3/4
I am using the last version of the docker image :
ghcr.io/cyberofficial/ttvdropbot/ttvdropbot-patched:latest

**Comment by Wyatt323** [comment]
i have the same question ， did you solve it？

**Comment by cyberofficial** [comment]
Currently under work!

**Comment by cyberofficial** [comment]
Small update, I might have found a way to fix this issue fully (for now) , i still need to do a bit more testing for stability and I need to make sure it works on linux. I'm using windows right now, and if it works on my linux set up then i'm golden and i'll push a beta update out for people test then ill collect feedback.

**Comment by delta-whiplash** [comment]
nice could you build a beta docker package and create a beta release

**Comment by johnwatso** [comment]
> Small update, I might have found a way to fix this issue fully (for now) , i still need to do a bit more testing for stability and I need to make sure it works on linux. I'm using windows right now, and if it works on my linux set up then i'm golden and i'll push a beta update out for people test then ill collect feedback.

Thanks for keeping this updated! would be happy to beta test too

**Comment by delta-whiplash** [comment]
hello I just found this repo I post it here if it could help
https://github.com/PockySweet/Enhanced-TTVDropBot
To resolve our issue

**Comment by tariqazmat101** [comment]
> Small update, I might have found a way to fix this issue fully (for now) , i still need to do a bit more testing for stability and I need to make sure it works on linux. I'm using windows right now, and if it works on my linux set up then i'm golden and i'll push a beta update out for people test then ill collect feedback.

Is it possible to disable this functionality instead? 
![image](https://user-images.githubusercontent.com/19142578/219231083-16acfa57-a11d-424e-a7e2-deade1f53562.png)

I can claim the drops myself.

**Comment by tariqazmat101** [comment]
> Small update, I might have found a way to fix this issue fully (for now) , i still need to do a bit more testing for stability and I need to make sure it works on linux. I'm using windows right now, and if it works on my linux set up then i'm golden and i'll push a beta update out for people test then ill collect feedback.

looking forward to the update :)

**Comment by johnwatso** [comment]
> > Small update, I might have found a way to fix this issue fully (for now) , i still need to do a bit more testing for stability and I need to make sure it works on linux. I'm using windows right now, and if it works on my linux set up then i'm golden and i'll push a beta update out for people test then ill collect feedback.
> 
> Is it possible to disable this functionality instead? ![image](https://user-images.githubusercontent.com/19142578/219231083-16acfa57-a11d-424e-a7e2-deade1f53562.png)
> 
> I can claim the drops myself.

Yes you can disable in the settings.json file "Autoclaim": False,

**Comment by cyberofficial** [comment]
I'm aware of this issue, fix is on the way in a few days.

Just making sure everything is passing tests to be considered okay to push out.
I really dislike pushing a fix that only works for small portion of people.

**Comment by xperisa** [comment]
> Small update, I might have found a way to fix this issue fully (for now) , i still need to do a bit more testing for stability and I need to make sure it works on linux. I'm using windows right now, and if it works on my linux set up then i'm golden and i'll push a beta update out for people test then ill collect feedback.

bro，is it fixed now？
